web/static/js/components/idea_edit_form.jsx

```diff
@@ -4,16 +4,23 @@ import * as AppPropTypes from "../prop_types"
 class IdeaEditForm extends Component {
   constructor(props) {
     super(props)
-    this.state = { ideaBody: props.idea.body }
+    this.state = {
+      ideaBody: props.idea.body,
+      ideaCategory: props.category,
+    }
     this.onChange = this.onChange.bind(this)
     this.onSubmit = this.onSubmit.bind(this)
     this.onCancel = this.onCancel.bind(this)
   }
 
   onChange(event) {
     const { retroChannel, idea } = this.props
-    retroChannel.push("idea_live_edit", { id: idea.id, liveEditText: event.target.value })
-    this.setState({ ideaBody: event.target.value })
+    if (event.target.name === "editable_idea") {
```

Comment:
rather than bundling everything into `onChange`, let's change this to `onChangeIdeaBody` and `onChangeIdeaCategory`. keeps them simpler

Reply:
sounds good - will do!